src/examples/memstats/memstats.go

```diff
@@ -0,0 +1,32 @@
+package main
```

Comment:
This example should be included in the Makefile (near the top of the `smoketest` target, with the other example smoke tests).

Reply:
✅